src/Samples/CSharp/Analyzers/CSharpAnalyzers/CSharpAnalyzers.Test/project.json

```diff
@@ -1,10 +1,7 @@
 {
-    "dependencies": {
-    },
-    "frameworks": {
-      "net46": {}
-    },
-    "runtimes": {
-      "win7": { }
-    }
+  "dependencies": {
+  },
+  "frameworks": {
+    "net46": { }
+  }
```

Comment:
It should allow deployment of implementation binaries.  Might not be necessary if we aren't copying them (or there aren't any since we only run the test on desktop)?